spec/build_command_generator_spec.rb

```diff
@@ -7,7 +9,11 @@
     end
 
     it "supports additional parameters" do
-      options = { project: "./example/standard/Example.xcodeproj", sdk: "9.0" }
+      xcargs_hash = { DEBUG: "1", BUNDLE_NAME: "Example App" }
```

Comment:
I think it would be nice to have and remove a lot of confusion (for fastlane users) to use a hash. Maybe in a future commit?